centreon/src/Core/Security/Authentication/Domain/Provider/OpenIdProvider.php

```diff
@@ -838,9 +816,9 @@ private function getUsernameFromLoginClaim(): string
             $this->getUserInformationFromUserInfoEndpoint();
         }
         if (! array_key_exists($loginClaim, $this->userInformations)) {
-            $this->centreonLog->insertLog(
-                CentreonUserLog::TYPE_LOGIN,
-                '[Openid] [Error] Unable to get login from claim: ' . $loginClaim
+            CentreonLog::create()->error(
+                CentreonLog::TYPE_LOGIN,
+                '[Openid] Unable to get login from claim: ' . $loginClaim
             );
             $this->error('Login Claim not found', ['login_claim' => $loginClaim]);
```

Comment:
two differents loggers, it's normal ?

Reply:
Yes, there is the LoggerTrait that is been used to write in centreon-web.log and CentreonLog to write in login.log

Reply:
ok thanks